tests/testthat/test-content.R

```diff
@@ -324,3 +362,34 @@ test_that("an error is raised when terminate_jobs() calls a bad URL", {
     )
   })
 })
+
+test_that("get_job_log() gets job logs", {
+  with_mock_api({
+    client <- Connect$new(server = "http://connect.example", api_key = "not-a-key")
+    item <- content_item(client, "8f37d6e0")
+    job_list <- get_job_list(item)
+    job <- purrr::keep(job_list, ~ .x$key == "mxPGVOMVk6f8dso2")[[1]]
+    log <- get_job_log(job)
```

Comment:
`key == "mxPGVOMVk6f8dso2"` is finding a specific job from the fixtures that you then assert about later, yeah? A comment might be nice that you're picking that one so that you can assert the shape of the data down below.

Reply:
I don't fully understand what you mean here, but I added a clarifying comment.